modules/nf-core/gatk4/markduplicates/gatk4-markduplicates.diff

```diff
@@ -0,0 +1,45 @@
+Changes in module 'nf-core/gatk4/markduplicates'
```

Comment:
Do the diff files need to be shipped with the module? I wasn't aware of it, but all good if they do!